src/lib/operators/jit_operator/jit_aware_lqp_translator.cpp

```diff
@@ -179,113 +184,46 @@ std::shared_ptr<JitOperatorWrapper> JitAwareLQPTranslator::_try_translate_sub_pl
   return jit_operator;
 }
 
-std::shared_ptr<const JitExpression> JitAwareLQPTranslator::_try_translate_subplan_to_jit_predicate_expression(
-    const std::shared_ptr<AbstractLQPNode>& node, JitReadTuples& jit_source,
-    const std::shared_ptr<AbstractLQPNode>& input_node) const {
-
-  switch (node->type) {
-    case LQPNodeType::Predicate: {
-      const auto left = _try_translate_predicate_node_to_jit_expression(std::dynamic_pointer_cast<PredicateNode>(node), jit_source,
-                                                 input_node);
-
-      // If the PredicateNode has further conditions (either PredicateNodes or UnionNodes) as children, we need a
-      // logical AND here. In this case, the condition represented by the PredicateNode itself becomes the left side
-      // and the input to the predicate node becomes the right side of the conjunction.
-      // If the PredicateNode has no further conditions (i.e., all tuples pass into the PredicateNode unconditionally),
-      // there is no need to create a dedicated AND-expression.
```

Comment:
This comment got lost, didn't it?

Reply:
I think it's ok to be deleted because the structure of the expressions changed.
Example expression: `a>1 and b<10`
Before, there was not a direct "and" expression. It was represented via two expressions were one expression was in the input to the other.
So the expression `a>1` was the input to the expression `b<10`.
With this pr there is an "and" expression which has the input to the expression`a>1` and the expression `b<10`.